### PR TITLE
CI: tag the published docker image as 'latest'

### PR DESCRIPTION
**Pull request by matklad**
Another regression introduced in the new release process! The current latest release is the one from a month ago. 

I am not fully familiar with how `:latest` is implemented in docker, so I am not sure that this is the right fix, but I've verified manually that at least passing `--tag` twice doesn't immediately return an error.

### CODE REVIEW

**Comment by KMJ-007** [comment]
waiting for this to get merged!

so can continue my work!

**Comment by matklad** [comment]
@KMJ-007 note that, even before this is merged, you can use explicit version of the container, `ghcr.io/tigerbeetle/tigerbeetle:0.14.158` rather than just `ghcr.io/tigerbeetle/tigerbeetle` (which is implicitly latest).

**Comment by KMJ-007** [comment]
cool

**Review by cb22 — APPROVED** [review]
> I am not fully familiar with how :latest is implemented in docker, so I am not sure that this is the right fix, but I've verified manually that at least passing --tag twice doesn't immediately return an error.

It's just a special tag by convention - tags in Docker aren't immutable. :upside_down_face:

**Pull request merged**
